spec/routes/db_spec.rb

```diff
@@ -0,0 +1,598 @@
+require 'spec_helper'
+require './app'
+
+describe App, type: :feature do
+  context 'GET /db' do
+    let(:paths) do
+      [
+        '/db/host_names',
+        '/db/asns',
+        '/db/ip_addresses',
+        '/db/mac_addresses',
+        '/db/open_ports',
+        '/db/ports',
+        '/db/services',
+        '/db/vulns',
+        '/db/urls',
+        '/db/url_schemes',
+        '/db/passwords',
+        '/db/credentials',
+        '/db/advisories',
+        '/db/software',
+        '/db/software_vendors',
+        '/db/oses',
+        '/db/phone_numbers',
+        '/db/street_addresses',
+        '/db/organizations',
+        '/db/people'
+      ]
+    end
+
+    it 'must return status 200 and include links to models' do
+      visit '/db'
+
+      expect(page.status_code).to eq(200)
+      paths.each do |path|
+        expect(page).to have_xpath("//a[@href='#{path}']")
```

Comment:
If one of the `paths` fails to show up in the page, then the whole spec fails. It would be better to move the `.each` loop outside of the `it` block.

```ruby
%w[
  ...
].each do |path|
  it "must contain a link to #{path}" do
    visit '/db'

    expect(page).to have_xpath("//a[@href='#{path}']")
  end
end
```